src/System.Net.Http.WinHttpHandler/ref/System.Net.Http.WinHttpHandler.csproj

```diff
@@ -1,9 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <ProjectGuid>{8C9AABA7-A8F0-4A5A-8A5F-65A0EFC03483}</ProjectGuid>
-    <Configurations>netstandard-Debug;netstandard-Release</Configurations>
+    <Configurations>netstandard-Debug;netstandard-Release;net461-Debug;net461-Release;netfx-Debug;netfx-Release</Configurations>
+    <!-- Ensure Assemblies are first resolved via targeting pack, and then using refpath -->
+    <AssemblySearchPaths>$(NuGetPackageRoot)\microsoft.targetingpack.netframework.v4.6.1\1.0.1\lib\net461\;$(AssemblySearchPaths)</AssemblySearchPaths>
```

Comment:
I actually just realized that I need an extra condition on this property to have this only happen for netfx builds and not netstandard, I will fix that